app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        __class__.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
```

Comment:
The 'create_person_list' function should take a list of dictionaries and return a list of 'Person' instances. However, there is no input type hint for the 'people' parameter. Please add a type hint for better code readability.